apps/studio/src/components/PageEditor/constants.ts

```diff
@@ -316,6 +321,12 @@ export const BLOCK_TO_META: Record<
     description: "Display dynamic data banner",
     usageText: "This block supports fetching data from an API endpoint.",
   },
+  logocloud: {
+    // TODO: Add image source
```

Comment:
when do we plan to add the image?

Reply:
when we need to add studio support! this is the image that shows on hover when adding the component on studio